src/common/src/DocumentTable/DocumentTable.tsx

```diff
@@ -65,13 +68,15 @@ const DocumentTable: React.FC<Props> = ({
 	pageSize = 10,
 	onRemove,
 	onDownload,
+	loading = false,
 }: Props) => {
 	const tableColumns = useMemo(() => {
 		let cols = columns ?? [
 			{
 				field: 'filename',
 				headerName: 'Bestandsnaam',
 				renderCell: function renderCell(params) {
+					if (loading) return <Skeleton height="20px" width="100px" />;
```

Comment:
Too bad that the inline styles need to be part of the hack. Isn't there a prettier way to get this done?

Reply:
Ah yes, these are actually redundant. I've removed them